routemaster/jobs/fail.rb

```diff
@@ -0,0 +1,15 @@
+require 'routemaster/mixins/log'
+
+module Routemaster
+  module Jobs
+    class Fail
+      include Mixins::Log
+
+      def call(*args)
+        raise 'failing job'
+      end
+    end
+  end
+end
+
```

Comment:
I can't see a nice way unless we manually require all the jobs in that directory at the top of the file for the `Job` class, what is the reason for not doing that, startup time?